communitiesuk/funding-service-design-fund-store
# FS-4323 Adding HSRA fund

https://dluhcdigital.atlassian.net/browse/FS-4323

### Change description
Adding fund config for HSRA



### How to test
- Run the script to load the new fund
- Alter the date in the database
The local link should be http://localhost:3008/funding-round/hsra/r1

### Screenshots of UI changes (if applicable)
